folly/AtomicLinkedList.h

```diff
@@ -67,7 +68,11 @@ class AtomicLinkedList {
   }
 
   bool empty() const {
+#ifdef MSVC_NO_NONVOID_ATOMIC_IF
+    return head_.load() == nullptr;
+#else
     return head_ == nullptr;
+#endif
```

Comment:
If we're supporting broken compilers, just call .load() explicitly for all compilers. And add a comment explaining why.